### ISSUE TITLE
Shortcut for referring to youself in third person

### ISSUE THREAD

**Issue by auroraanna** [nextcloud/spreed#10321]
**Is your feature request related to a problem? Please describe.**
When chatting, some people prefer to speak in third person when referring to themself.
- Examples
  - first person
    1. I went grocery shopping because I was missing vegetables for dinner.
    2. I invited my friend to come over for a sleepover.
  - third person
    1. Anna went grocery shopping because she was missing vegetables for dinner.
    2. She invited her friend to come over for a sleepover.

This can be ambiguous when using their name, because there might be several persons in the same chat, with the same name. If you want to write about yourself in third person, writing you name might become tedious if it's long or difficult to type.

The name however, might be required in order to give context when using third person pronouns.

**Describe the solution you'd like**
A shortcut e.g. something like typing `/me` in the chat which is then transformed into a reference to the user's display name or a shortened version of it, if it's long.

**Describe alternatives you've considered**
A shortcut to mention yourself.

**Additional context**
None.

**Comment by nickvergessen** [comment]
Yeah, always wanted to improve `/me` behaviour but it's currently overlapping with commands.
Maybe now that we plan to drop them in favor of bots, we could do a string replacement
